robolectric/src/test/java/org/robolectric/android/internal/AndroidTestEnvironmentTest.java

```diff
@@ -111,10 +117,45 @@ public void setUpApplicationState_setsMainThread_onAnotherThread() throws Interr
    * in Robolectric via {@link BouncyCastleProvider}.
    */
   @Test
-  public void ensureBouncyCastleInstalled() throws GeneralSecurityException {
+  @ConscryptMode(ON)
+  public void testWhenConscryptModeOn_ConscryptInstalled()
+      throws CertificateException, NoSuchAlgorithmException {
 
     bootstrapWrapper.callSetUpApplicationState();
+    CertificateFactory factory = CertificateFactory.getInstance("X.509");
+    assertThat(factory.getProvider().getName()).isEqualTo("Conscrypt");
 
+    MessageDigest digest = MessageDigest.getInstance("SHA256");
+    assertThat(digest.getProvider().getName()).isEqualTo("Conscrypt");
+  }
+
+  @Test
+  @ConscryptMode(ON)
+  public void testWhenConscryptModeOn_BouncyCastleInstalled() throws GeneralSecurityException {
+
+    bootstrapWrapper.callSetUpApplicationState();
+    Cipher aesCipher = Cipher.getInstance("RSA/None/OAEPWithSHA-256AndMGF1Padding");
```

Comment:
I remember @hoisie has a question about this value at another old closed PR.

Reply:
@utzcoz it was on this PR itself in `line no. 162` and I have already changed it. Let me check for this one, if that works then I will update it.

Reply:
Some tests failed when I changed it to `AES/CBC/PKCS7Padding`.

Reply:
Is `AES/CBC/PKCS7Padding` supported by Conscrypt?